charts/opentelemetry-demo/Chart.yaml

```diff
@@ -11,6 +11,6 @@ maintainers:
 appVersion: "0.0.1"
 dependencies:
   - name: opentelemetry-demo
-    version: 0.2.1
+    version: 0.1.4
```

Comment:
mis-changes? why degrade?

Reply:
The latest helm chart uses 0.3.4-alpha as images version, some service still have errors,I want to use the stable one to make sure generate traces.